SETUP/tests/phpunit_bootstrap.php

```diff
@@ -1,8 +1,16 @@
 <?php
 global $relPath;
+global $ACCESS_CRITERIA; // HACK see https://github.com/sebastianbergmann/phpunit/issues/325
```

Comment:
The better fix is to explicitly make these global in the files they are defined rather than implicitly here. This is a general problem in our code where we assume that if we define variables outside of a function they are global -- but they aren't. We've had to make these types of fixes before.

Reply:
Thanks, I'll do that.